### ISSUE TITLE
Have compared the inplace_sync_abn with the mit sync bn?

### ISSUE THREAD

**Issue by ycszen** [PkuRainBow/OCNet#20]
Have you used this [sync-bn](https://github.com/vacancy/Synchronized-BatchNorm-PyTorch)? And have you compared the speed and the final performance with this two sync-bn?

**Comment by PkuRainBow** [comment]
You are welcome to report the numbers based on the BN of other versions.

**Comment by Liuyixuan95** [comment]
@PkuRainBow 
I tried ysnc-bn but it will stop in certain iteration without any notion output . The utilization rate of GPU will drop to 0 and the memory useage are still full.  I tried 8 TITAN, 6 TITAN, 4 TITAN to train, all failed.
But when I used inplace -abn it seems normal....

**Comment by PkuRainBow** [comment]
@Liuyixuan95 So have you reproduced the results?

In fact, I just employ the inplace-abn and it seems that the inplace-abn does not support 8x gpu cards.

We are preparing a new work that enable you to train the OCNet with 4x Pascal Titan GPUs.

Please keep waiting for our new work.  We will try to release the paper by the end of the Dec.

**Comment by denru01** [comment]
Same issue here. I have 6 Titan X (Pascal) and sync-inplace-bn only works when 2 GPUs are used. 4 or 6 GPUs do not work.
